vms/platformvm/txs/executor/standard_tx_executor_test.go

```diff
@@ -1997,39 +2009,6 @@ func TestStandardExecutorRemoveSubnetValidatorTx(t *testing.T) {
 			},
 			expectedErr: ErrFlowCheckFailed,
 		},
-		{
-			name: "attempted to remove subnet validator after subnet conversion has occurred",
-			newExecutor: func(ctrl *gomock.Controller) (*txs.RemoveSubnetValidatorTx, *StandardTxExecutor) {
-				env := newValidRemoveSubnetValidatorTxVerifyEnv(t, ctrl)
-				env.state.EXPECT().GetSubnetConversion(env.unsignedTx.Subnet).Return(
-					state.SubnetConversion{
-						ConversionID: ids.GenerateTestID(),
-						ChainID:      ids.GenerateTestID(),
-						Addr:         []byte("address"),
-					},
-					nil,
-				).AnyTimes()
-				env.state.EXPECT().GetTimestamp().Return(env.latestForkTime).AnyTimes()
-
-				cfg := &config.Config{
-					UpgradeConfig: upgradetest.GetConfigWithUpgradeTime(upgradetest.Etna, env.latestForkTime),
-				}
-				e := &StandardTxExecutor{
-					Backend: &Backend{
-						Config:       cfg,
-						Bootstrapped: &utils.Atomic[bool]{},
-						Fx:           env.fx,
-						FlowChecker:  env.flowChecker,
-						Ctx:          &snow.Context{},
-					},
-					Tx:    env.tx,
-					State: env.state,
-				}
-				e.Bootstrapped.Set(true)
-				return env.unsignedTx, e
-			},
-			expectedErr: ErrRemoveValidatorManagedSubnet,
```

Comment:
Is it possible to retain the test with `expectedErr` as nil instead?

Reply:
The first test in the vector was modified to do this